packages/orbit-components/src/primitives/TooltipPrimitive/TooltipPrimitive.stories.tsx

```diff
@@ -188,7 +202,7 @@ export const WithImageInside: Story = {
 };
 
 export const Playground: Story = {
-  render: ({ children, content, ...args }) => (
+  render: ({ children, content, "aria-labelledby": ariaLabelledby, ...args }) => (
```

Comment:
I was thinking, maybe we can fix the stories a bit (applies to all 3 files) - since it's not recommended to use interactive elements inside the tooltip, we would remove the link from there. And we could also either remove the stories with tooltip on icon or add aria- attributes there as it makes more sense to have it there than here in Playground where it's already accessible with text. WDYT?

Reply:
yes, I was also considering to align the stories based on the updated tooltip and our observation.